lightning-network-tools/lnd/payments.md

```diff
@@ -133,7 +133,7 @@ This section covers the use of the _SendPaymentV2_ API endpoint, which finds a r
 
 ### Payment Fees
 
-Nodes in the Lightning Network that route payments on behalf of other nodes charge fees in exchange for the use of their capital. Each node in the network is free to set their fees as they see fit. If you send a multi-hop payment \(ie, a payment that is not to one of the nodes you have a direct channel with\), your will need to pay fees per-hop that the payment uses.
+Nodes in the Lightning Network that route payments on behalf of other nodes charge fees in exchange for the use of their capital. Each node in the network is free to set their fees as they see fit. If you send a multi-hop payment \(ie, a payment that is not to one of the nodes you have a direct channel with\), you will need to pay fees per-hop that the payment uses.
```

Comment:
I think the original intention was probably to say "your node"?

Reply:
I think that is more accurate, yes.